src/Dialect/ONNX/Rewrite.td

```diff
@@ -312,6 +317,22 @@ def RemoveIdentityReshapePattern:  Pat<
   // Check that val has the specified shape.
   [(HasSpecifiedConstantShape $val, $shape)]>;
 
+def RemoveReshapeBeforeMatmul : Pattern<
```

Comment:
Suggests to modify the name of this pattern to `RemoveReshapeBracketingMatMul` because the pattern expects 2 Reshape operations 'around' a matrix multiply operation (rather than just one before it).

Reply:
no longer relevant given the new implementation.